src/screencastkeys/ops.py

```diff
@@ -188,6 +188,27 @@ class ScreencastKeysStatus(bpy.types.Operator):
         EventType.OSKEY
     ]
 
+    mouse_event_types = {
+        EventType.LEFTMOUSE,  # Left Mouse
+        EventType.MIDDLEMOUSE,  # Middle Mouse
+        EventType.RIGHTMOUSE,  # Right Mouse
+        EventType.BUTTON4MOUSE,  # Button4 Mouse
+        EventType.BUTTON5MOUSE,  # Button5 Mouse
+        EventType.BUTTON6MOUSE,  # Button6 Mouse
+        EventType.BUTTON7MOUSE,  # Button7 Mouse
+        # EventType.PEN,  # Pen
+        # EventType.ERASER,  # Eraser
+        # EventType.MOUSEMOVE,  # Mouse Move
+        # EventType.INBETWEEN_MOUSEMOVE,  # In-between Move
```

Comment:
Why these events are commented out?  
If these events are not necessary, please delete them.

Reply:
Alright, they're gone! I didn't know what they did, so I left them in in case they were useful. The documentation for events isn't very good...